### ISSUE TITLE
intellisense for $event in the html Template doesn't work

### ISSUE THREAD

**Issue by mhamri** [angular/vscode-ng-language-service#531]
**Describe the bug**
$event. trigger the wrong set of suggestion for the autocomplete

![image](https://user-images.githubusercontent.com/8991783/71476679-349da580-2821-11ea-9284-271877c0d6e8.png)

based on the event that in here is (click) the $event should suggest correct `MouseEvent` items, or on (blur) should suggest `FocuseEvent` etc.

I expect to see, `stopImmediatePropagation()` and other items such in here.

**Logs**

Please attach two logs:

1. Console output (Go to toolbar --> View --> Output, and copy the contents)
```
[Info  - 5:12:00 PM] Angular language server process ID: 21952
[Info  - 5:12:00 PM] Using typescript v3.6.4 from c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js
[Info  - 5:12:00 PM] Using @angular/language-service v9.0.0-rc.7 from c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js
[Info  - 5:12:00 PM] Log file: c:\Users\MohammadHosseinAmri\AppData\Roaming\Code\logs\20191226T162205\exthost1\Angular.ng-template\nglangsvc.log
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
Warning: Can't resolve all parameters for ProjectsService in c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts: (?). This will become an error in Angular v6.x
[Error - 7:36:45 PM] TypeError: Cannot read property 'startRecordingFilesWithChangedResolutions' of undefined
    at ConfiguredProject.Project.updateGraph (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:129744:38)
    at ConfiguredProject.updateGraph (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:130316:63)
    at updateProjectIfDirty (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:130674:45)
    at Array.forEach (<anonymous>)
    at ProjectService.assignProjectToOpenedScriptInfo (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:132622:41)
    at ProjectService.openClientFileWithNormalizedPath (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:132652:31)
    at ProjectService.openClientFile (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:132536:29)
    at ProjectService.openClientFile (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\index.js:811:32)
    at Session.onDidOpenTextDocument (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\index.js:1047:49)
    at c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\index.js:937:47
[Error - 7:36:45 PM] Notification handler 'textDocument/didOpen' failed with message: Cannot read property 'startRecordingFilesWithChangedResolutions' of undefined
c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:129744
                this.resolutionCache.startRecordingFilesWithChangedResolutions();
                                     ^

TypeError: Cannot read property 'startRecordingFilesWithChangedResolutions' of undefined
    at ConfiguredProject.Project.updateGraph (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:129744:38)
    at ConfiguredProject.updateGraph (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:130316:63)
    at Object.updateProjectIfDirty (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:130674:45)
    at ConfiguredProject.Project.getLanguageService (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js:129429:28)
    at Session.sendPendingDiagnostics (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\index.js:1009:35)
    at Timeout._onTimeout (c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\index.js:990:19)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
[Info  - 7:37:01 PM] Connection to server got closed. Server will restart.
[Info  - 7:37:01 PM] Angular language server process ID: 23912
[Info  - 7:37:01 PM] Using typescript v3.6.4 from c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\node_modules\typescript\lib\tsserverlibrary.js
[Info  - 7:37:01 PM] Using @angular/language-service v9.0.0-rc.7 from c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js
[Info  - 7:37:01 PM] Log file: c:\Users\MohammadHosseinAmri\AppData\Roaming\Code\logs\20191226T162205\exthost1\Angular.ng-template\nglangsvc.log

```

2. Log file (Path is printed to the console output at startup)
```
Info 0    [19:37:1.471] Format host information updated
Info 1    [19:37:1.473] reload projects.
Info 2    [19:37:1.474] Structure before ensureProjectForOpenFiles:
Info 3    [19:37:1.474] Structure after ensureProjectForOpenFiles:
Info 4    [19:37:1.474] Host file extension mappings updated
Info 5    [19:37:1.490] Search path: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/src/app/projects
Info 6    [19:37:1.492] For info: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/src/app/projects/projects.component.ts :: Config file name: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 7    [19:37:1.492] Opened configuration file c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 8    [19:37:1.571] Loading global plugin @angular/language-service
Info 9    [19:37:1.571] Enabling plugin @angular/language-service from candidate paths: c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 10   [19:37:1.572] Loading @angular/language-service from c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/node_modules/node_modules)
Info 11   [19:37:1.598] Loading @angular/language-service from c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js (resolved to c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/server/node_modules/@angular/language-service/bundles/language-service.umd.js/node_modules)
Info 12   [19:37:1.725] Plugin validation succeded
Info 13   [19:37:1.767] Starting updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 14   [19:37:8.757] Finishing updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json Version: 1 structureChanged: true Elapsed: 6990ms
Info 15   [19:37:8.757] Project 'c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json' (Configured) 
Info 16   [19:37:8.757] 	Files (576)

Info 17   [19:37:8.757] -----------------------------------------------
Info 18   [19:37:8.760] Starting updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 19   [19:37:8.784] Finishing updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json Version: 2 structureChanged: false Elapsed: 24ms
Info 20   [19:37:8.784] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 21   [19:37:8.801] Search path: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects
Info 22   [19:37:8.805] For info: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/projects.service.ts :: Config file name: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json
Info 23   [19:37:8.805] Opened configuration file c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json
Info 24   [19:37:8.894] Loading global plugin @angular/language-service
Info 25   [19:37:8.894] Enabling plugin @angular/language-service from candidate paths: c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 26   [19:37:8.894] Loading @angular/language-service from c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/node_modules/node_modules)
Info 27   [19:37:8.897] Loading @angular/language-service from c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js (resolved to c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/server/node_modules/@angular/language-service/bundles/language-service.umd.js/node_modules)
Info 28   [19:37:8.939] Plugin validation succeded
Info 29   [19:37:8.947] Starting updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json
Info 30   [19:37:9.657] Finishing updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json Version: 1 structureChanged: true Elapsed: 710ms
Info 31   [19:37:9.657] Project 'c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json' (Configured) 
Info 32   [19:37:9.657] 	Files (104)

Info 33   [19:37:9.657] -----------------------------------------------
Info 34   [19:37:9.660] Starting updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json
Info 35   [19:37:9.673] Finishing updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json Version: 2 structureChanged: false Elapsed: 13ms
Info 36   [19:37:9.673] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 37   [19:37:9.687] Search path: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects
Info 38   [19:37:9.687] For info: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/projects/project.ts :: Config file name: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json
Info 39   [19:37:9.699] Search path: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src
Info 40   [19:37:9.699] For info: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/index.ts :: Config file name: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json
Info 41   [19:37:9.700] Search path: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib
Info 42   [19:37:9.700] For info: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/src/lib/core-data.module.ts :: Config file name: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/libs/core-data/tsconfig.json
Info 43   [19:37:9.701] Search path: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/src/app
Info 44   [19:37:9.701] For info: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/src/app/app.module.ts :: Config file name: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 45   [19:37:9.703] Search path: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/src/app/projects
Info 46   [19:37:9.704] For info: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/src/app/projects/projects.component.html :: Config file name: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 47   [19:37:9.705] Loading global plugin @angular/language-service
Info 48   [19:37:9.705] Enabling plugin @angular/language-service from candidate paths: c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 49   [19:37:9.705] Loading @angular/language-service from c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/node_modules/node_modules)
Info 50   [19:37:9.708] Loading @angular/language-service from c:\Users\MohammadHosseinAmri\.vscode\extensions\angular.ng-template-0.900.3\server\node_modules\@angular\language-service\bundles\language-service.umd.js (resolved to c:/Users/MohammadHosseinAmri/.vscode/extensions/angular.ng-template-0.900.3/server/node_modules/@angular/language-service/bundles/language-service.umd.js/node_modules)
Info 51   [19:37:9.732] Plugin validation succeded
Info 52   [19:37:9.745] Starting updateGraphWorker: Project: /dev/null/inferredProject1*
Info 53   [19:37:11.217] Finishing updateGraphWorker: Project: /dev/null/inferredProject1* Version: 1 structureChanged: true Elapsed: 1472ms
Info 54   [19:37:11.217] Project '/dev/null/inferredProject1*' (Inferred) 
Info 55   [19:37:11.217] 	Files (29)

Info 56   [19:37:11.217] -----------------------------------------------
Info 57   [19:37:11.227] Search path: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/src/app/projects
Info 58   [19:37:11.228] For info: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/src/app/projects/projects.component.html :: Config file name: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 59   [19:37:11.228] Starting updateGraphWorker: Project: /dev/null/inferredProject1*
Info 60   [19:37:11.262] Finishing updateGraphWorker: Project: /dev/null/inferredProject1* Version: 2 structureChanged: false Elapsed: 34ms
Info 61   [19:37:11.262] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 62   [19:37:14.80] Starting updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 63   [19:37:14.126] Finishing updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json Version: 3 structureChanged: false Elapsed: 46ms
Info 64   [19:37:14.126] Project 'c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json' (Configured) 
Info 65   [19:37:14.127] 	Files (576)

Info 66   [19:37:14.127] -----------------------------------------------
Info 67   [19:37:42.324] Starting updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 68   [19:37:42.347] Finishing updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json Version: 4 structureChanged: false Elapsed: 23ms
Info 69   [19:37:42.347] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 70   [19:37:44.819] Starting updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 71   [19:37:44.858] Finishing updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json Version: 5 structureChanged: false Elapsed: 39ms
Info 72   [19:37:44.858] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 73   [19:37:45.398] Starting updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
Info 74   [19:37:45.432] Finishing updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json Version: 6 structureChanged: false Elapsed: 34ms
Info 75   [19:37:45.432] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 76   [19:38:0.530] Starting updateGraphWorker: Project: c:/Users/MohammadHosseinAmri/source/repos/learning/angular-core-workshop/apps/dashboard/tsconfig.json
```

[edit(ayazhafiz)]: shortened extra entries of log file

**Comment by ayazhafiz** [comment]
Thank you for the bug report; this has been a long standing todo. I will try to get to it today.

**Comment by kyliau** [comment]
This is fixed by https://github.com/angular/angular/pull/34570

**Comment by ayazhafiz** [comment]
Not quite done; we are missing typings for DOM events. Please see the attached PR; we may decide to actually not support DOM event typing.

**Comment by kyliau** [comment]
This has been fixed by the new Ivy-native language service, released in [v11.1.0](https://github.com/angular/vscode-ng-language-service/releases/tag/v11.1.0).
It's an opt-in feature for now, please give it a try and let us know if you have any feedback.
For the best editor experience, please make sure your project has [`strictTemplates`](https://angular.io/guide/angular-compiler-options#stricttemplates) enabled in `angularCompilerOptions`.
In case you run into similar bug, please file a new issue. I'll close this for now.
